packages/foundation-models/src/azure-openai/azure-openai-chat-completion-stream-chunk-response.test.ts

```diff
@@ -0,0 +1,77 @@
+import { parseMockResponse } from '../../../../test-util/mock-http.js';
+import { AzureOpenAiChatCompletionStreamChunkResponse } from './azure-openai-chat-completion-stream-chunk-response.js';
+
+describe('OpenAI chat completion stream chunk response', () => {
+  let mockResponses: {
+    tokenUsage: any;
+    finishReason: any;
+    deltaContent: any;
+  };
+  let azureOpenAiChatCompletionStreamChunkResponses: {
+    tokenUsage: AzureOpenAiChatCompletionStreamChunkResponse;
```

Comment:
[q] Why are each of these properties of type `AzureOpenAiChatCompletionStreamChunkResponse`. Should they be part of this type?

Reply:
I think this is a bit confusing. What I want is to have three chunk responses, and `tokenUsage` is like one of the chunk response which has a token usage property.

Reply:
I would update this to make it easy to understand. In the current state it is a bit misleading

Reply:
Indeed misleading. I changed them. Thanks!